lib/rules/selector-combinator-space-after/__tests__/index.js

```diff
@@ -135,79 +136,136 @@ testRule(rule, {
     {
       code: "namespace|type#id > .foo {}",
       description: "qualified id with namespace"
+    },
```

Comment:
I'm confused by these tests are they aren't valid CSS. Do we need them?

Reply:
I didn't know it was invalid css...
I removed this test case.